cmake/SetupPch.cmake

```diff
@@ -151,9 +151,9 @@ if (USE_PCH)
       )
   endif()
 
-  # Override the default add_library and add_executable function provided
-  # by CMake so that it adds the precompiled header as a dependency to
-  # all of them.
+  # Override the default add_library and add_spectre_executable function
+  # provided by CMake so that it adds the precompiled header as a
```

Comment:
This reads like `add_spectre_executable` is provided by CMake, which I don't think is true.. perhaps rephrase?